<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Command</class>
 <widget class="QGroupBox" name="Command">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <widget class="QComboBox" name="commandBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>120</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>150</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="copyButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>520</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearButton">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>520</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="resetButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>100</y>
     <width>150</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="categoryBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>30</y>
     <width>120</width>
     <height>32</height>
    </rect>
   </property>
   <property name="currentText">
    <string>git command</string>
   </property>
   <item>
    <property name="text">
     <string>git command</string>
    </property>
   </item>
  </widget>
  <widget class="QTextEdit" name="textEdit_3">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>100</y>
     <width>150</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>641</width>
     <height>271</height>
    </rect>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="optionsBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>120</width>
     <height>32</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>options</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="resetButton_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Generate command</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
